api/v1/api.go

```diff
@@ -27,38 +29,64 @@ func NewApiV1() *apiV1 {
 // @type apiKey
 // @in header
 // @name Authorization
-func (s *apiV1) RegisterRoutes(e *echo.Echo) {
-	e.GET("/status", s.handleStatus)
-
+func RegisterRoutes(e *echo.Echo, config config.TenantServiceConfig) {
+	apiGroup := e.Group("/api/v1")
+	e.Use(middleware.RequestID())
 	e.Use(AuthMiddleware)
 
-	// /collections
-	e.POST("/collections", s.handleCreateCollection)
-	e.GET("/collections", s.handleGetCollections)
-	e.PUT("/collections", s.handleModifyCollection)
-	e.DELETE("/collections", s.handleDeleteCollection)
+	ConfigureStatusRouter(apiGroup, config)
+}
+
+func GetTenantId(c echo.Context) int {
+	return int(c.Get(TENANT_CONTEXT).(AuthContext).TenantID)
+}
 
-	// /storage-contract
-	e.GET("/storage-contract", s.handleGetStorageContract)
-	e.POST("/storage-contract", s.handleSetStorageContract)
+func UnixToFilEpoch(unixEpoch int64) int64 {
```

Comment:
its fine for now since its just one function, but in future it would be good to break this out into a separate sub-package in the repo called `utils` or similar